2/contrib/openshift/base-plugins.txt

```diff
@@ -29,9 +31,22 @@ openshift-client:1.0.35
 openshift-login:1.0.26
 openshift-sync:1.0.46
 pam-auth:1.6
+parameterized-trigger:2.36
+pipeline-build-step:2.12
+pipeline-github-lib:1.0
+pipeline-input-step:2.11
+pipeline-model-api:1.7.2
+pipeline-model-definition:1.6.0
+pipeline-utility-steps:2.5.0
```

Comment:
Note - these are the current plugins for Jenkins on OCP 4.6.

Reply:
I think I need to drop these plugins.

Per https://github.com/openshift/jenkins/issues/1292#issuecomment-864016370 openshift-client pulls in these dependencies implicitly. We should not hard-pin the dependency.

Reply:
Not all of these were implicitly pulled in by other plugins. This broke our upgrade path from 4.7 -> 4.8 Jenkins containers. Open issue #1342